### PR TITLE
Show favourites

### PR DESCRIPTION
**Pull request by Swaathik**

### CODE REVIEW

File: src/components/Title/index.js [hunk 1]
```diff
@@ -314,22 +336,57 @@ class Title extends PureComponent /*:: <Props> */ {
...
+                    role="button"
+                    onClick={() => this.manageFavourites(metadata)}
+                    onKeyDown={() => this.manageFavourites(metadata)}
+                    tabIndex={0}
+                  >
+                    <span
+                      className={f(
+                        'icon',
+                        'icon-common',
+                        this.props.entries.includes(metadata.accession)
```
Comment (gustavo-salazar) [inline src/components/Title/index.js:367]: Changed this way as I think it's a bit more react-like...
this way the dom element doesn't change only its css class.

Although I'm not actually sure if the virtual DOM takes care of it

File: src/components/home/FavouriteEntries/index.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+                {this.state.fav.map((f) => (
+                  <EntryCard entry={f} key={f.metadata.accession} />
+                ))}
+              </AnimatedEntry>
+            </div>
+          </div>
+        </div>
+      );
+    }
+    return (
```
Comment (gustavo-salazar) [inline src/components/home/FavouriteEntries/index.js:78]: Just added a bit more text to inform about how to favourite entries

File: src/storage/idb/index.js [hunk 3]
```diff
@@ -18,6 +19,9 @@ const init = () => {
         case 0:
           db.createObjectStore(IPScanJobsMeta, { autoIncrement: true });
           db.createObjectStore(IPScanJobsData, { autoIncrement: true });
+        // eslint-disable-next-line no-fallthrough
+        case 1:
```
Comment (gustavo-salazar) [inline src/storage/idb/index.js:23]: So this way, when someone that already has a version of InterPro running, gets only the update of the version 2.

**Review by gustavo-salazar — APPROVED** [review]

File: src/components/Title/index.js (hunk 1 above)
Comment (Swaathik) [reply src/components/Title/index.js:367]: Oh right. I was just getting rid of the errors and warnings at the time which I forgot to refactor it later.

**Pull request merged**